### DISCUSSION TITLE
Revamp the web design

### DISCUSSION THREAD

**Discussion by mgilangjanuar** [mgilangjanuar/teledrive#50] [Ideas]
Hello everyone, my friend @Muhammad-0704 made something cool here please take a look @grgp @naufaldi @Aqshola 🤣

![Home](https://user-images.githubusercontent.com/3146378/136961331-f14cb2dc-31c3-43bf-8d6a-fda91adb04e7.png)

**Comment by Aqshola** [comment]
WOAAAAAAAAAAH ITS AWESOMEEE, well im agree if we use this 😎👍

**Comment by mgilangjanuar** [comment]
Good, now I need help implementing this into our project 🤣

**Reply by Aqshola** [reply]
Share the figma link and lets gooo🚀🚀

**Reply by mgilangjanuar** [reply]
Cool! cc @Muhammad-0704

**Reply by Muhammad-0704** [reply]
Okay, I'll try to finish it up today. And then we'll drown👍

**Comment by naufaldi** [comment]
is the design finished??

**Comment by mgilangjanuar** [comment]
Hello guys @naufaldi @Aqshola here's what we're waiting for 🥳 https://www.figma.com/file/l6ko5GHYaeDbnMQlpzXoMz/TeleDrive-web-site-v.-2?node-id=0%3A1 

by @Muhammad-0704

**Comment by mgilangjanuar** [comment]
Oh and, here's our new logo by @hanifibadur https://github.com/mgilangjanuar/teledrive/tree/staging/web/public/teledrive-logo 🚀

**Comment by Aqshola** [comment]
WOAAAAHHHH its awesooomeeee

**Comment by Aqshola** [comment]
Well i think it would look like this?
![image](https://user-images.githubusercontent.com/52363719/142729482-ecf3dda2-fcc9-4858-9430-f46042216987.png)

**Reply by mgilangjanuar** [reply]
Cool 🥰
